FandomParser/FandomParser/WikiBuildingInfoProvider.cs

```diff
@@ -65,17 +67,15 @@ private static WikiBuildingInfo ParseWikiBuildingInfo(WikiTextTableEntry table)
                 else if (!string.IsNullOrWhiteSpace(table.Size))
                 {
                     var splittedSize = table.Size.Split('x');
-                    if (!int.TryParse(splittedSize[0], out int x))
-                    {
-
-                    }
 
-                    if (!int.TryParse(splittedSize[1], out int y))
+                    var couldParseX = int.TryParse(splittedSize[0], out int x);
+                    var couldParseY = int.TryParse(splittedSize[1], out int y);
+                    if (!couldParseX || !couldParseY)
                     {
-
+                        logger.Warn($"could not parse Size for \"{result.Name} ({result.Region})\": \"{table.Size}\"");
```

Comment:
Minor thing - you could use 2 double quotes `""` with `@` to escape the quotes in the string. e.g
`$@"could not parse Size for ""{result.Name} ({result.Region})"": ""{table.Size}""");`

Reply:
The current style matches the rest of the whole repo and I think this is more common in general.